docs/how-to/integrations/craft-a-ruby-app.rst

```diff
@@ -36,8 +36,9 @@ Add a part written in Ruby
     :start-at: parts:
     :end-at: - git
 
-Ruby parts are built with the `Ruby <https://snapcraft.io/docs/ruby-plugin>`_
-plugin.
+.. Missing link to Ruby plugin
```

Comment:
Unrelated: ruby is only available for core and core18 so this page shouldn't have made it into the new docs.

Reply:
https://github.com/canonical/snapcraft/pull/5530/commits/7875bb5d73b2dd0bedda263b8fb11e99d446d8a6